lib/spack/spack/test/make_executable.py

```diff
@@ -10,12 +10,18 @@
 """
 import os
 import shutil
+import sys
 import tempfile
 import unittest
 
+import pytest
+
 from spack.build_environment import MakeExecutable
 from spack.util.environment import path_put_first
 
+pytestmark = pytest.mark.skipif(sys.platform == "win32",
+                                reason="does not run on windows")
```

Comment:
I assume that there is also no intent to get this running on Windows (since `MakeExecutable` is only for `make` as it runs on Linux/MacOS/Unix systems)